etc/sysconfig/pacemaker.in

```diff
@@ -275,6 +275,16 @@
 #
 # Default: PCMK_ipc_buffer="131072"
 
+# PCMK_cib_timeout (Advanced Use Only)
```

Comment:
Alternatively this could go to its own section. It's technically related to inter-process communication but isn't part of the IPC configuration.